csvcubed/tests/behaviour/steps/inspectcli.py

```diff
@@ -19,13 +19,19 @@ def _unformat_multiline_string(string: str) -> str:
 
 @When('the existing Metadata file exists "{csvw_metadata_file}"')
 def step_impl(context, csvw_metadata_file: str):
-    print(csvw_metadata_file)
     context.csvw_metadata_json_path = (
         get_context_temp_dir_path(context) / csvw_metadata_file
     )
     path = Path(context.csvw_metadata_json_path)
-    assert path.exists() == True
+    assert path.exists() is True
 
+@When('the existing csv file exists "{csv_file}"')
+def step_impl(context, csv_file: str):
```

Comment:
I thought we already had a behaviour step to assert that a file exists? 

https://github.com/GSS-Cogs/csvcubed/blob/main/csvcubed-devtools/csvcubeddevtools/behaviour/file.py#L68